Jetchat/app/src/main/res/drawable/ic_launcher_monochrome.xml

```diff
@@ -0,0 +1,43 @@
+<!--
+  ~ Copyright 2020 The Android Open Source Project
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="110dp"
```

Comment:
You are right. I just copied the `ic_launcher_foreground` though, which already had the bug.
I just changed the size without changing the path coordinates, because it's a tiny change.

Reply:
The dangers of touching existing code 😆